rules/50-pandectas.rules

```diff
@@ -0,0 +1,2 @@
+KERNEL=="hiddev*", SUBSYSTEM=="usbmisc", ATTRS{idVendor}=="13ff", ATTRS{idProduct}=="0008", GROUP="adm", MODE=0664"
+KERNEL=="uinput", MODE="0660", GROUP="adm", OPTIONS+="static_node=uinput"
```

Comment:
I don't know what the `static_node=uinput` part means, but assuming it does not mitigate the following problem, I believe this opens a hole for any flatpak running under a user in the adm group to be able to implement a virtual input device. Such a device could perform like a regular input peripheral, and take over control of the user's cursor, for example.

From your comment on the issue I understand you have looked for alternatives, but isn't this to big of a whole that we are opening here? Can we somehow make this 2nd rule more restrictive? If I'm not mistaken, the default user created during FBE is part of the admin group.